modules/rosa-create-objects.adoc

```diff
@@ -273,6 +273,84 @@ Create a cluster with autoscaling enabled on the default worker machine pool.
 $ rosa create cluster --cluster-name=mycluster -region=us-east-1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
 
+[id="rosa-create-external-auth-provider_{context}"]
+== create external-auth-provider
+
+Add an external identity provider instead of the OpenShift OAuth2 server.
+
+[IMPORTANT]
+====
+You can only use external authentication providers on {hcp-title} clusters.
+====
+
+.Syntax
+[source,terminal]
+----
+$ rosa create external-auth-provider --cluster=<cluster_name> | <cluster_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--claim-mapping-groups-claim <string>     
+|Required. Describes rules on how to transform information from an ID token into a cluster identity.
+
+|--claim-validation-rule <strings>
+|Rules that are applied to validate token claims to authenticate users. The input will be in a <claim>:<required_value> format. To have multiple claim validation rules, you can separate the values by ','. For example, <claim>:<required_value>,<claim>:<required_value> format.
+
+|--claim-mapping-username-claim <string>
+|The name of the claim that should be used to construct user names for the cluster identity.
+
+a|--cluster <cluster_name>\|<cluster_id>
+|Required. The name or ID of the cluster the IDP will be added.
+
+|--console-client-id <string>
+|The identifier of the OIDC client from the OIDC provider for the OpenShift web console.
+
+|--console-client-secret <string>          
+|The secret that is associated with the console application registration.
+
+|--issuer-audiences <strings>
+|An array of audiences to check the incoming tokens against. Valid tokens must include at least one of these values in their "aud" claim.
+
+|--issuer-ca-file <string>
+|The path to the PEM-encoded certificate file to use when making requests to the server.
+
+|--issuer-url <string>
+|The serving URL of the token issuer.
+
+|--name <string>
+|A name that is used to refer to the external authentication provider.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Specifies an AWS profile string from your credentials file.
+|===
+
+.Examples
+Add an Entra ID identity provider to a cluster named `mycluster`.
```

Comment:
Since it's the first appearance, maybe use the full product name here:
```suggestion
Add a Microsoft Entra ID identity provider to a cluster named `mycluster`.
```